doc/man1/req.pod

```diff
@@ -394,10 +394,10 @@ option. For compatibility B<encrypt_rsa_key> is an equivalent option.
 
 =item B<default_md>
 
-This option specifies the digest algorithm to use.
-Any digest supported by the OpenSSL B<dgst> command can be used.
-If not present then MD5 is used.
-This option can be overridden on the command line.
+This option specifies the digest algorithm to use. Any digest supported by the
+OpenSSL B<dgst> command can be used. This option can be overridden on the
+command line. Certain signing algorithms (i.e. Ed25519 and Ed448) will ignore
+any digest that has been set.
```

Comment:
This is a behavior change, should it warrant an HISTORY entry?

Reply:
Is it really a behaviour change? There aren't any algorithms in 1.1.0 where this clause applies. You gave the example somewhere else of someone providing an engine to do ed25519 in 1.1.0. I'm not convinced that is actually possible. In 1.1.0 we assume that all signing algorithms require a digest...and stuff will probably break if you try to supply one via an engine where that isn't true.

Reply:
I commented on the behavior change explicitly because I noticed that it started breaking stuff for an engine I am working on ([libsuola](https://github.com/romen/libsuola)) which does exactly this, and I noticed that after this change 1.1.1 apps have the option of ignoring explicitly set configuration entries or cmdline args, while 1.1.0 apps don't.

I just wanted to make sure that all the implications of this change are discussed before committing to them! ;)

Reply:
> I noticed that it started breaking stuff for an engine I am working on

Interesting. Can you be more explicit about in what way it broke?